library/functions.po

```diff
@@ -2819,16 +2829,14 @@ msgstr ""
 "informations générales à propos des chaînes, consultez :ref:`textseq`."
 
 #: ../Doc/library/functions.rst:1579
-#, fuzzy
 msgid ""
 "Sums *start* and the items of an *iterable* from left to right and returns "
 "the total.  The *iterable*'s items are normally numbers, and the start value "
 "is not allowed to be a string."
 msgstr ""
-"Additionne *start* et les éléments d'*iterable* de gauche à droite et en "
-"donne le total. *start* vaut ``0`` par défaut. Les éléments d'*iterable* "
-"sont normalement des nombres, et la valeur de *start* ne peut pas être une "
-"chaîne."
+"Additionne *start* et les éléments d'un *iterable* de gauche à droite et en "
+"donne le total. Les éléments d'*iterable* sont normalement des nombres, et "
```

Comment:
```suggestion
"renvoie le total. Les éléments d'\ *iterable* sont normalement des nombres, et "
```
je pense qu'il faut échapper l'apostrophe devant le marquant `*`.

Reply:
Je trouve que 'donne' passe mieux dans ce contexte

Reply:
> je pense qu'il faut échapper l'apostrophe devant le marquant *.

Ça ne passe pas avec powrap

Reply:
> je pense qu'il faut échapper l'apostrophe devant le marquant *.

Ça ne passe plus avec pospell. Je rajoute dans le dictionnaire ou je traduit itérable entre les *?
Je laisse sans l'échappement pour l'instant.

Reply:
```suggestion
"donne le total. Les éléments de `iterable` sont normalement des nombres, et "
```
iterable étant un paramètre de la fonction, on peut l'écrire ainsi.
ou alors « les éléments de l'itérable » doit passer.